projects/go-lib/src/lib/components/go-header/go-header.component.html

```diff
@@ -0,0 +1,22 @@
+<header class="go-header">
+  <div class="go-header__left"
+       [ngClass]="{ 'go-header__left--collapsed': !sideNavService.navOpen }">
+    <go-icon class="go-header__menu"
```

Comment:
This should be wrapped in a `<button>` tag where the click action is on the button.

Reply:
should we just use a go-icon-button here instead?

Reply:
@StevenUlmer yes, good idea. I've updated this.